packages/components/src/BrandMoment/BrandMoment.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from "react"
+import { render } from "@testing-library/react"
+import { BrandMomentError } from "@kaizen/draft-illustration"
+import { Paragraph } from "@kaizen/typography"
+import { BrandMoment } from "./BrandMoment"
+import { Box } from "./components/Box"
+
+// eslint-disable-next-line ssr-friendly/no-dom-globals-in-module-scope
+window.matchMedia = jest.fn().mockImplementation(() => ({
+  matches: false,
+  media: "",
+  onchange: null,
+  addListener: jest.fn(),
+  addEventListener: jest.fn(),
+  removeListener: jest.fn(),
+  removeEventListener: jest.fn(),
+}))
+
+const mockPlay = jest.fn().mockResolvedValue(undefined)
+const mockLoad = jest.fn()
+const mockPause = jest.fn()
+
+describe("<BrandMoment />", () => {
+  beforeEach(() => {
+    window.HTMLMediaElement.prototype.load = mockLoad
+    window.HTMLMediaElement.prototype.play = mockPlay
+    window.HTMLMediaElement.prototype.pause = mockPause
+    Object.defineProperty(HTMLMediaElement.prototype, "muted", {
+      set: jest.fn(),
+    })
+  })
+
+  it("matches the snapshot", () => {
```

Comment:
Just kill this file. It's only a snapshot test, and you didn't even include it lol